chromium_src/chrome/browser/upgrade_detector/upgrade_detector_impl.cc

```diff
@@ -5,6 +5,7 @@
 
 #include "build/build_config.h"
 
+#if BUILDFLAG(IS_MAC)
```

Comment:
leaving this inside `IS_MAC` defeats the purpose here. The point is to control this by the feature flag, not an os flag. If anything maybe this should go behind `ENABLE_UPDATER`?